src/Compilers/CSharp/Test/Semantic/Semantics/OperatorTests.cs

```diff
@@ -40,7 +39,51 @@ static void Main()
         Console.Write(rn);
     }
 }";
-            CompileAndVerify(source: source, expectedOutput: "TwoZero");
+            CompileAndVerify(source: source, expectedOutput: "TwoZero")
+                .VerifyOperationTree(@"
+Test.Main()
+  IBlockStatement (8 statements, 6 locals) (OperationKind.BlockStatement)
+    Local_1: e (Type: E)
+    Local_2: x (Type: int)
+    Local_3: r (Type: E)
+    Local_4: en (Type: E?)
+    Local_5: xn (Type: int?)
+    Local_6: rn (Type: E?)
```

Comment:
What happens if something changes and the baseline doesn't match? How easy would it be to figure out where the difference and how to adjust the baseline?